standards/SPIFFE_Workload_API.md

```diff
@@ -10,51 +10,43 @@ Portable and interoperable cryptographic identity for networked workloads is per
 
 ## Table of Contents
 
-1\. [Introduction](#1-introduction)  
-2\. [Extensibility](#2-extensibility)  
-3\. [Service Defintion](#3-service-definition)  
-4\. [Identifying the Caller](#4-identifying-the-caller)  
-5\. [X.509-SVID Profile](#5-x509-svid-profile)  
-5.1. [Workload API Client and Server Behavior](#51-workload-api-client-and-server-behavior)  
-5.2. [Federated Bundles](#52-federated-bundles)  
-5.3. [Default Identity](#53-default-identity)  
-5.4. [Profile Messages](#54-profile-messages)  
-5.5. [Default Values and Redacted Information](#55-default-values-and-redacted-information)  
-Appendix A. [Sample Implementation State Machines](#appendix-a-sample-implementation-state-machines)  
+1\. [Introduction](#1-introduction)
+2\. [Extensibility](#2-extensibility)
+3\. [Service Defintion](#3-service-definition)
+4\. [Identifying the Caller](#4-identifying-the-caller)
+5\. [X.509-SVID Profile](#5-x509-svid-profile)
+5.1. [Workload API Client and Server Behavior](#51-workload-api-client-and-server-behavior)
+5.2. [Federated Bundles](#52-federated-bundles)
+5.3. [Default Identity](#53-default-identity)
+5.4. [Profile Definition](#54-profile-definition)
+5.5. [Default Values and Redacted Information](#55-default-values-and-redacted-information)
+6\. [JWT-SVID Profile](#6-jwt-svid-profile)
+6.1 [Client and Server Behavior](#61-client-and-server-behavior)
+6.2 [Default Identity](#62-default-identity)
+6.3 [Fetching Bundles](#63-fetching-bundles)
+6.4 [Profile Definition](#64-profile-definition)
+6.5. [Default Values and Redacted Information](#65-default-values-and-redacted-information)
 
 ## 1. Introduction
 
 The SPIFFE Workload API is an API which provides information and services that enable workloads, or compute processes, to leverage SPIFFE identities and SPIFFE-based authentication systems. It is served by the [SPIFFE Workload Endpoint](SPIFFE_Workload_Endpoint.md), and comprises a number of services, or *profiles*.
 
-Currently, only one profile is defined: the [X.509-SVID Profile](#5-x509-svid-profile). As such, supporting this profile is mandatory. Future versions of this specification may introduce additional profiles, and the X.509-SVID Profile may become optional.
+Currently, there are two profiles, both of which are mandatory:
```

Comment:
> both of which are mandatory

I couldn't remember where the group landed in terms of optionality. Happy to amend this if it isn't accurate.

Reply:
@evan2645 reminded me of what we discussed in SIG-SPEC a bit ago, that the profile is mandatory but can be administratively disabled.